src/App.jsx

```diff
@@ -1,8 +1,27 @@
 import React from 'react';
+import styles from './App.module.css';
 
 class App extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = { buttonColor: 'red' };
+    this.handleCLick = this.handleCLick.bind(this);
+  }
+
+  handleCLick() {
+    this.setState((state) => {
+      const color = state.buttonColor === 'red' ? 'blue' : 'red';
```

Comment:
really stange code: if smth == red then blue else red. I hope you can do it better